curvefs/src/mds/common/mds_define.h

```diff
@@ -39,6 +44,26 @@ typedef uint32_t UserIdType;
 
 const uint32_t UNINTIALIZE_ID = 0u;
 
+inline std::string BuildPeerId(const std::string &ip, uint32_t port,
+                               uint32_t idx = 0) {
+    return ip + ":" + std::to_string(port) + ":" + std::to_string(idx);
+}
+
+inline bool SplitPeerId(const std::string &peerId, std::string *ip,
+                        uint32_t *port, uint32_t *idx = nullptr) {
+    std::vector<std::string> items;
+    SplitString(peerId, ":", &items);
+    if (3 == items.size()) {
+        *ip = items[0];
+        *port = std::stoul(items[1]);
```

Comment:
careful about it, it can throw exceptions if the string doesn't store a valid value or value is out of range.
use https://github.com/opencurve/curve/blob/a0b92d3680c5125bb66f8ad1a6deb33624446288/src/common/string_util.h#L72 or define another `StringToUl`

Reply:
> careful about it, it can throw exceptions if the string doesn't store a valid value or value is out of range.
> use
> 
> https://github.com/opencurve/curve/blob/a0b92d3680c5125bb66f8ad1a6deb33624446288/src/common/string_util.h#L72
> 
> or define another `StringToUl`

Add StringToUl.